PracticeTemplate/Controllers/Ex6/Ex6ViewController.swift

```diff
@@ -10,8 +10,50 @@ import UIKit
 
 class Ex6ViewController: BaseViewController {
     
+    // MARK: - IBoulet
+    @IBOutlet private weak var grayView: UIView!
+    @IBOutlet private weak var blueView: UIView!
+    @IBOutlet private weak var thumbView: UIView!
+    @IBOutlet private weak var thumbLabel: UILabel!
+    
     override func viewDidLoad() {
```

Comment:
Nếu ở `BaseViewController` có khai báo `setupUI` rồi thì chỉ cần **override** lại `setupUI` thôi, vì `setupUI` cũng được gọi ở viewDidLoad trong `BaseViewController`
```suggestion
    override func setupUI() {
```